src/index.html

```diff
@@ -9,11 +9,38 @@
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
+  <title>Layout product card</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="http://yui.yahooapis.com/3.18.1/build/cssreset/cssreset-min.css">
 </head>
 <body>
-
-<script type="text/javascript" src="./main.js"></script>
+  <div class="card" data-qa="card">
+    <a href="#" class="card-image-link">
+      <img class="card-image" src="./images/imac.jpeg" alt="APPLE A1419 iMac 27" title="APPLE A1419 iMac 27">
+    </a>
+    <h1 class="card-describe">
```

Comment:
`<h1>` defines the most important heading. `<h6>` defines the least important heading.
In this situation you need `<h2>` or `<h3>`